src/compression/bytes32.rs

```diff
@@ -0,0 +1,34 @@
+use std::fmt::{Debug, Formatter};
```

Comment:
@arvid I know you already have a `Bytes32` type in this crate, so this is far from ideal. I'll try to migrate over to yours, but it's not a drop-in replacement as this one is basically a tuple with one entry `[u8; 32]`, and the existing one is a typedef.

Reply:
Done.